crowbar_framework/app/helpers/barclamp/neutron_helper.rb

```diff
@@ -17,35 +17,15 @@
 
 module Barclamp
   module NeutronHelper
+
     def networking_plugins_for_neutron(selected)
       options_for_select(
-        [
-          ["ml2", "ml2"],
-          ["vmware", "vmware"]
-        ],
+        NeutronService.networking_plugins_valid.map{|el| [el, el]},
```

Comment:
Done in https://github.com/crowbar/barclamp-neutron/commit/ed72a9a35db400e62101fa774ed32947ee5880b1